### PR TITLE
Run Frontend's tests separately

### PR DESCRIPTION
**Pull request by kzys**
Run Frontend's tests and Backend's tests in parallel.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,16 +48,26 @@ matrix:
...
-        - nvm install 10
-        - npm ci
         - rustup component add rustfmt-preview
         - rustup component add clippy-preview
       script:
         - cargo fmt -- --check
         - cargo clippy --all-targets --all-features --all
         - cargo test
+    - name: Frontend
+      rust: stable
```
Comment (Turbo87) [inline .travis.yml]: do we need `rust` here at all? 

```yaml
language: node_js
node_js: 10
```
 should automatically invoke `nvm install`

File: .travis.yml [hunk 2]
```diff
@@ -48,16 +48,26 @@ matrix:
...
+      before_install:
+        - nvm install 10
+        - npm ci
```
Comment (Turbo87) [inline .travis.yml]: this should probably be moved to the `install` step while we're at it

File: .travis.yml [hunk 3]
```diff
@@ -48,16 +48,26 @@ matrix:
...
+      # skip diesel
+      install:
+        - echo
+      before_script:
+        - echo
```
Comment (Turbo87) [inline .travis.yml]: why the `echo` lines?

File: .travis.yml (hunk 1 above)
Comment (kzys) [reply .travis.yml]: Does it work. I thought that the `language:` wouldn't be applied at the Build Matrix level.

File: .travis.yml (hunk 2 above)
Comment (kzys) [reply .travis.yml]: Right. Then we can remove `echo` on install:

File: .travis.yml (hunk 3 above)
Comment (kzys) [reply .travis.yml]: To override the top-level install/before_script: but we probably can get rid of most of them. Let me update the PR.

File: .travis.yml (hunk 1 above)
Comment (kzys) [reply .travis.yml]: Travis CI's documentation regarding using multiple languages also doesn't use `language:` on Build Matrix.

https://docs.travis-ci.com/user/customizing-the-build#installing-a-second-programming-language

Comment (Turbo87) [reply .travis.yml]: https://github.com/skylines-project/skylines/blob/43a68499581ea0ad04ee70e40a6f3a1f49683928/.travis.yml#L81-L82

seems to work though 😉

File: .travis.yml (hunk 3 above)
Comment (Turbo87) [reply .travis.yml]: you can use `install: skip` in those cases (see https://docs.travis-ci.com/user/build-stages/#build-stages-and-deployments)

Comment (kzys) [reply .travis.yml]: ohh, didn't know that.

**Comment by locks** [comment]
oops xD

**Pull request merged**